pkg/sync/v5/data/ConfigMap/openshift-azure-logging/fluentd-config.yaml

```diff
@@ -7,6 +7,7 @@ data:
         @type local
         path /var/log/journald.pos
       </storage>
+      read_from_head true
```

Comment:
we might also want to add `read_from_head` [here](https://github.com/openshift/openshift-azure/blob/5951f298a5ce20d01b9f06eabb99b7b29a8dfc31/pkg/sync/v5/data/ConfigMap/openshift-azure-logging/fluentd-config.yaml#L19)?

Reply:
I read the docs (and the code) and believe that we shouldn't - I think that if you specify multiple files fluentd is supposed to "do the right thing" - am going to test manually